script/deploy/mainnet/005_RegisterLidoWithdrawalsScript.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.23;
+
+import "forge-std/console.sol";
+import {Vm} from "forge-std/Vm.sol";
+
+import {IERC20, IWETH, LegacyAMM} from "contracts/Interfaces.sol";
+import {LidoARM} from "contracts/LidoARM.sol";
+import {CapManager} from "contracts/CapManager.sol";
+import {Proxy} from "contracts/Proxy.sol";
+import {ZapperLidoARM} from "contracts/ZapperLidoARM.sol";
+import {Mainnet} from "contracts/utils/Addresses.sol";
+import {GovProposal, GovSixHelper} from "contracts/utils/GovSixHelper.sol";
+import {AbstractDeployScript} from "../AbstractDeployScript.sol";
+
+contract RegisterLidoWithdrawalsScript is AbstractDeployScript {
+    using GovSixHelper for GovProposal;
+
+    GovProposal public govProposal;
+
+    string public constant override DEPLOY_NAME = "005_RegisterLidoWithdrawalsScript";
+    bool public constant override proposalExecuted = false;
+
+    Proxy lidoARMProxy;
+    Proxy capManProxy;
+    LidoARM lidoARMImpl;
+    LidoARM lidoARM;
+    CapManager capManager;
+    ZapperLidoARM zapper;
```

Comment:
Not used:
- lidoARMProxy
- capManProxy
- lidoARM
- capManager
- zapper

Reply:
I've removed these. Thanks